hermes-api/src/main/java/pl/allegro/tech/hermes/api/MaintainerDescriptor.java

```diff
@@ -0,0 +1,53 @@
+package pl.allegro.tech.hermes.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+public final class MaintainerDescriptor {
```

Comment:
maybe we should change then name to MaintainerId and it will be easier? i know `source` does not fit `Id`, but i would argue it will be nicer to see `MaintainerId` all over place, not to wonder what `Descriptor` really is